xbmc.py

```diff
@@ -283,22 +283,24 @@ def play(self, item=None, listitem=None, windowed=False, startpos=-1):
         self.onAVStarted()
 
     def isPlaying(self):  # NOSONAR
-        # type: () -> bool
-        """
-        Check Kodi is playing something.
+        """ Check Kodi is playing something.
 
         :return: True if Kodi is playing a file.
+        :rtype: bool
         """
         return self.play_started
 
-    def onAVStarted(self):  # NOSONAR
-        # type: () -> None
+    def getPlayingFile(self):  # NOSONAR
+        """ Returns the current playing file as a string.
+
+        :return: The filename of the playing item.
+        :rtype: str
         """
-        onAVStarted method.
+        return Player.playing_file if self.play_started else None
```

Comment:
I think the `Player.playing_file` is not defined with a default value anywhere. So calling `getPlayingFile` before `setRevolvedUrl` will fail. So  `Player.playing_file` should be defined somewhere here:

```python 

class Player(KodiStub):
    playing_file = None

    def __init__(self, *args, **kwargs):
        super(Player, self).__init__()

```

Reply:
That's right!

It would probably also make sense to set `playing_file` when using the `play()` method of the `Player`, but I don't use this anywhere in my code, so I'll leave that up to somebody else in the future, along with some other unimplemented methods in the `Player`.